### PR TITLE
Update Flutter Getting Sarted tutorial

### PR DESCRIPTION
**Pull request by lohanidamodar**
Based on suggestion
https://github.com/appwrite/appwrite/issues/1599

### CODE REVIEW

File: app/views/docs/getting-started-for-flutter.phtml [hunk 1]
```diff
@@ -33,21 +33,18 @@ $version = (isset($versions['flutter'])) ? $versions['flutter'] : '';
...
 
-<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your <a href="https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
+<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added below `application` tag along side existing `activity` tags to your <a href="https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
```
Comment (abnegate) [inline app/views/docs/getting-started-for-flutter.phtml:36]:
````suggestion
<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added inside the `application` tag, along side the existing `activity` tags in your <a href="https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
```

File: app/views/docs/getting-started-for-flutter.phtml [hunk 2]
```diff
@@ -33,21 +33,18 @@ $version = (isset($versions['flutter'])) ? $versions['flutter'] : '';
...
-      <intent-filter android:label="flutter_web_auth">
-        <action android:name="android.intent.action.VIEW" />
-        <category android:name="android.intent.category.DEFAULT" />
-        <category android:name="android.intent.category.BROWSABLE" />
-        <data android:scheme="appwrite-callback-[PROJECT_ID]" />
-      </intent-filter>
-    </activity>
-  </application>
-</manifest>
+    <pre class="line-numbers"><code class="prism language-xml" data-prism><?php echo $this->escape('<!-- Add this between `application` tag along side existing `activity` tags -->
```
Comment (abnegate) [inline app/views/docs/getting-started-for-flutter.phtml:39]:
```suggestion
    <pre class="line-numbers"><code class="prism language-xml" data-prism><?php echo $this->escape('<!-- Add this inside the `application` tag, along side the existing `activity` tags -->
```

**Review by christyjacob4 — APPROVED** [review]
Looks good.

File: app/views/docs/getting-started-for-flutter.phtml [hunk 3]
```diff
@@ -33,21 +33,18 @@ $version = (isset($versions['flutter'])) ? $versions['flutter'] : '';
...
+<activity android:name="com.linusu.flutter_web_auth.CallbackActivity" >
```
Comment (2002Bishwajeet) [inline app/views/docs/getting-started-for-flutter.phtml:40]: How about writing Like this?
```xml
<manifest ...... >
.
.
<application ................ >
.
.
.
<!-- Add inside the existing application tag along the side of existing activity tags  ---->
<activity android:name="com.linusu.flutter_web_auth.CallbackActivity" >
<intent-filter android:label="flutter_web_auth">
    <action android:name="android.intent.action.VIEW" />
    <category android:name="android.intent.category.DEFAULT" />
    <category android:name="android.intent.category.BROWSABLE" />
    <data android:scheme="appwrite-callback-[PROJECT_ID]" />
  </intent-filter>
</activity>
.
.
.
.
</application>
.
.
.
</manifest>
```
That's how I was thinking . What do you guyz think?

Comment (eldadfux) [reply app/views/docs/getting-started-for-flutter.phtml:40]: Maybe with `...` instead of multiple `.` with line breaks

Comment (2002Bishwajeet) [reply app/views/docs/getting-started-for-flutter.phtml:40]: Yeah just something like that.  Couldn't express properly myself though😅

**Comment by lohanidamodar** [comment]
@abnegate Though android devs understand this better, after this is accepted, we might wanna update the android one following the same convention.

File: app/views/docs/getting-started-for-flutter.phtml (hunk 3 above)
Comment (lohanidamodar) [reply app/views/docs/getting-started-for-flutter.phtml]: @2002Bishwajeet Updated, do check.

**Review by 2002Bishwajeet — APPROVED** [review]
Awesome✨

**Comment by abnegate** [comment]
> @abnegate Though android devs understand this better, after this is accepted, we might wanna update the android one following the same convention.

Sure thing will update Android 👍

**Review by abnegate — APPROVED** [review]
Looks good

File: app/views/docs/getting-started-for-flutter.phtml [hunk 4]
```diff
@@ -33,11 +33,15 @@ $version = (isset($versions['flutter'])) ? $versions['flutter'] : '';
...
 
-<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your <a href="https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
+<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added inside the `<application>` tag, along side the existing `<activity>` tags in your <a href="https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
+```
```
Comment (abnegate) [inline app/views/docs/getting-started-for-flutter.phtml:37]:
```suggestion

```

**Pull request merged**